### ISSUE TITLE
Advanced Respawn Support

### ISSUE THREAD

**Issue by JonCup** [tim-savage/SavageDeathCompass#15]
I'm using Advanced Respawn (formerly known as GTA Respawn) When I die, the compass doesn't point at my death location, but the location I respawn at.

**Comment by tim-savage** [comment]
This plugin listens for the standard bukkit player death and player respawn events. When the player death event is fired, this plugin writes the player location to its datastore. When a player respawns, and the respawn was caused by a player death, it sets the player compass target to the location from its datastore (whose world matches player's current world). The other plugin must be doing something to disrupt this simple chain of events. There's really nothing I can do to work around this, the fix would need to be implemented in the other plugin.

tldr; They're not compatible and there's nothing I can do about it. Perhaps the developer of the other plugin can incorporate the functionality of this plugin within the other.
